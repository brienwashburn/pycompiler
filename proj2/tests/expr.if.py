10 if a or b else 30
