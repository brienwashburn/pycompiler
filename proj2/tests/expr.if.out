(Module (Expr (IfExp (BoolOp Or (Name a) (Name b)) (Num 10) (Num 30))))
